### ISSUE TITLE
integer constructor strange missingValueValue

### ISSUE THREAD

**Issue by jspaaks** [NLeSC/enram#74]
[code](https://github.com/NLeSC/enram/blob/294227a28cf79a94832d20eccb363915a717e184/ncradar/lib/vol2bird/libvol2bird.c#L944)

**Comment by adokter** [comment]
The missing value 255 was used as a nodata flag when writing images to files, see
https://github.com/NLeSC/enram/blob/master/doc/vol2bird-adriaans-version-20140716/vol2birdprof_h5.c#L239
and
https://github.com/NLeSC/enram/blob/master/doc/vol2bird-adriaans-version-20140716/vol2birdprof_h5.c#L831-L841

"nodata" was used for data outside the range where the radar was scanning and collecting data
"missing" was used to indicate gates where the radar detected no signal above the noise level

**Comment by jspaaks** [comment]
Yes, and using a value of 255 when the data type is ``unsigned char`` makes sense. However, this issue specifically questions the validity of assigning a missingValueValue of 255 to an array of ``int`` (even when it is not a problem in reality, since this is a case that seems to never actually occur).
